.nix/libgit2.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, python3, zlib, libssh2
+, openssl, pcre, http-parser, libiconv, Security, staticBuild ?
+  stdenv.hostPlatform.isStatic
+  # for passthru.tests
+, libgit2-glib, python3Packages }:
+
+stdenv.mkDerivation rec {
+  pname = "libgit2";
+  version = "1.7.2";
```

Comment:
is there a reason that the libgit2 1.7.1 in nixpkgs is not sufficient?

Reply:
pkg-config for openssl insists on 1.7.2

Reply:
for openssl? might be worth instead downgrading the libgit2 dep in the lockfile. 1.7.2 was released exactly two days ago.